### ISSUE TITLE
sorting objects by coords

### ISSUE THREAD

**Issue by SamKacer** [tspivey/pokecrystal-access#3]
One thing I was thinking while going down rute 32 on my new run. is how it is difficult to efficiently battle all the trainers, pick up items, pick trees, etc. in order while going down a new route.

What I found myself doing, since route 32 is north to south, cycling through the list of objects and going down to the next object that is the least number down from me. This way I will approximately take the most efficient route, like I would if I could see the screen.

However, the issue is that the list of objects isnt sorted, so I would have to cycle through the whole thing when I wanted to find the next thing to go to. This would be made much easier if the user could switch between the list being sorted  by type of object, as it is now, and it being sorted either east-west or north-south.

I know this wouldn't be useful for every single map, but it would be useful for most of them.

**Comment by ddugovic** [comment]
I agree; maybe have an option to sort by direction the player is facing.  [volume warning] https://clips.twitch.tv/AttractiveSpunkyAlmondGivePLZ

For example, a player at X facing East could hear `C D E B A` because C is closest to the right, and B is closer than A to the left:

```
A
B  X  C D
   E
```
